jenkins/shared-library/vars/monorepoSeedPipeline.groovy

```diff
@@ -69,8 +66,8 @@ void call(Closure body) {
 	}
 }
 
-String resolveBuildConfigurationFile(String configurationFile)  {
-	String filepath = Paths.get('.github').resolve(configurationFile)
+String resolveBuildConfigurationFile()  {
+	filepath = helper.resolveBuildConfigurationFile()
```

Comment:
does this pass groovy linter? i thought you would need explicit type?

Reply:
yea, it pass linter.
I will update it with the type.